lib/src/app.dart

```diff
@@ -11,39 +13,53 @@ import '../src/screens/signup_screen.dart';
 import '../src/services/user_services.dart';
 import 'screens/login_screen.dart';
 import 'screens/onboarding_screen.dart';
+import 'screens/user_profile_screen/my_profile_screen.dart';
+import 'screens/user_profile_screen/view_profile_screen.dart';
 
 class App extends StatelessWidget {
   final userServices = UserServices();
-
+  late Size designSize;
   @override
   Widget build(BuildContext context) {
     SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
       statusBarColor: Colors.transparent,
       statusBarBrightness: Brightness.dark,
       statusBarIconBrightness: Brightness.dark,
     ));
-    return MultiBlocProvider(
-      providers: [
-        BlocProvider(
-            create: (context) =>
-                ForgotPasswordBloc(userServices: userServices)),
-        BlocProvider(
-            create: (context) => LoginBloc(userServices: userServices)),
-        BlocProvider(
-            create: (context) => SignupBloc(userServices: userServices)),
-      ],
-      child: MaterialApp(
-        theme: ThemeData(fontFamily: "Nunito-Regular"),
-        debugShowCheckedModeBanner: false,
-        routes: {
-          "/": (context) => OnboardingScreen(),
-          "/forgotPasswordScreen": (context) => ForgotPasswordScreen(),
-          "/homeScreen": (context) => HomeScreen(),
-          "/loginScreen": (context) => LoginScreen(),
-          "/onboardingScreen": (context) => OnboardingScreen(),
-          "/signupScreen": (context) => SignupScreen(),
-        },
-      ),
-    );
+
+    if (Device.get().isPhone) {
+      designSize = Size(375, 812);
+    } else {
+      designSize = Size(770, 1024);
+    }
```

Comment:
We should move these to constants and at the same time, note why the values are set the way they are.